cms/djangoapps/contentstore/management/commands/reindex_course.py

```diff
@@ -22,32 +21,23 @@ class Command(BaseCommand):
 
     Examples:
 
-        ./manage.py reindex_course <course_id_1> <course_id_2> - reindexes courses with keys course_id_1 and course_id_2
+        ./manage.py reindex_course <course_id_1> <course_id_2> ... - reindexes courses with provided keys
         ./manage.py reindex_course --all - reindexes all available courses
         ./manage.py reindex_course --setup - reindexes all courses for devstack setup
     """
     help = dedent(__doc__)
-
     can_import_settings = True
-
-    args = "<course_id course_id ...>"
-
-    all_option = make_option('--all',
-                             action='store_true',
-                             dest='all',
-                             default=False,
-                             help='Reindex all courses')
-
-    setup_option = make_option('--setup',
-                               action='store_true',
-                               dest='setup',
-                               default=False,
-                               help='Reindex all courses on developers stack setup')
-
-    option_list = BaseCommand.option_list + (all_option, setup_option)
-
     CONFIRMATION_PROMPT = u"Re-indexing all courses might be a time consuming operation. Do you want to continue?"
 
+    def add_arguments(self, parser):
+        parser.add_argument('course_ids', nargs="*")
```

Comment:
Is this going to show in the help text like `<course_ids ...>`?  Should it just be `course_id` instead?